hasher-matcher-actioner/hmalib/matchers/matchers_base.py

```diff
@@ -123,7 +166,8 @@ def match(
         index = self.get_index(signal_type)
 
         with metrics.timer(metrics.names.indexer.search_index):
-            match_results: t.List[IndexMatch] = index.query(signal_value)
+            match_results: t.List[IndexMatch] = []
```

Comment:
Not following this change. If you are creating a list and then immediately extending it, why not use the `index.query(...)` result directly?

Reply:
oops refactor error